content/en/docs/reference/kubernetes-api/ref/common-definitions/podaffinity-.md

```diff
@@ -0,0 +1,73 @@
+---
+api_metadata:
+  apiVersion: ""
+  import: "k8s.io/api/core/v1"
+  kind: "PodAffinity"
+collapsible: false
+content_type: "api_reference"
+description: "Pod affinity is a group of inter pod affinity scheduling rules."
+draft: false
+title: "PodAffinity"
+weight: 16
+---
+
+
+
+`import "k8s.io/api/core/v1"`
+
+
+## PodAffinity
+
+Pod affinity is a group of inter pod affinity scheduling rules.
+
+
+- **preferredDuringSchedulingIgnoredDuringExecution** ([]WeightedPodAffinityTerm)
```

Comment:
The `WeightedPodAffinityTerm` should be a link.
I mean, for all type references, we should use links.

Reply:
wait ... are you expanding the `WeightedPodAffinityTerm` directly inline?

Reply:
Yes, it is documented inline